CollateralAccount/CollateralAccount.md

```diff
@@ -0,0 +1,106 @@
+1. Although the refactor transaction can be created under the hood for in-software wallet, it will confused users when it shows up on their transaction history. For hardware wallets, it will require explicit approval from the user which is also possibly confusing.
+1. Altough wallets can try and always pick UTXOs to make sure there is always a valid UTXO entry for collateral, it can't be guaranteed because the wallet state can always be changed by any dApp or other wallet that doesn't use the same logic to guarantee the presence of satisfactory collateral input
+1. Multisig wallets don't have any non-script addresses in general and so would need some separate solution
+
+So with this, we can see the naive solution is complicated while still having issues and an unintuitive user experience.
+
+
+
+# Specification
+
+Recall that [CIP1852](../CIP-1852) specifies the following derivation path
+
+```
+m / purpose' / coin_type' / account' / chain / address_index
+```
+
+We set `chain=3` to indicate the *collateral chain*.
+
+## *address_index* value
+
+Wallets MUST only use `address_index=0` for this specification. Since a single address can contain multiple UTXO entries, there is no need to derive other addresses (using more addresses would not provide any privacy benefit and would complicate address discovery).
+
+We will call this specific derivation the *collateral account*.
+
+## New solution for managing collateral
+
+Wallets SHOULD only use UTXO entries of the collateral account as collateral in the transaction. If the collateral is insufficient, the wallet should tell the user to send more ADA to their collateral account.
+
+The benefits of this solution are as follows:
+1. Transactions in the history to add collateral can be clearly marked as such by wallet software.
+1. Prompting the user to add ADA to their collateral account doesn't require explaining users about how UTXO works under the hood compared to the refactoring option.
+1. No change required to input selection algorithms
+1. Multisig (or other script-based wallets) are handled the same way as regular wallets
+1. User only risks as much collateral as their are comfortable with
+
+However, this solution also comes with downsides:
+
+1. Reusing the same collateral account for all your smart contract calls gets rid of any privacy. However, base addresses in Cardano already forgo privacy and so users who want to call smart contracts with privacy will already need to handle things differently.
+1. The blockchain can't stop people from sending tokens to somebody's collateral account. This doesn't harm the user (since other UTXO entries can still be used) and wallets will already need to implement a way for the user to withdraw from their collateral account so the impact should be minimal.
```

Comment:
Should this happen you would likely have to pay a small fee to transfer from collateral to main account? Whatever the protocol minimum parameters dictate?

Reply:
Yes, that's correct